tests/test_models_qcnn.py

```diff
@@ -13,6 +13,7 @@
 
 def test_classifier_circuit2():
     """ """
+    set_backend("numpy")
```

Comment:
Following the discussion above, this is also not very good practice because it will also set numpy as the `GlobalBackend` for all upcoming tests. However, since everything should work with numpy, it will probably not cause any issue. Maybe we could just open an issue for the test discussion and postpone for now.

Reply:
Yeah, I agree, however the `QCNN` model doesn't seem to support the specification of the execution backend and always uses the `GlobalBackend`, I opened an issue about this already #1362. The tests were originally using the `NumpyBackend` (the global one) and thus I set it explicitely here.